packages/bpk-component-map/src/BpkPriceMarker.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { cssModules } from 'bpk-react-utils';
+import ArrowDownIcon from 'bpk-component-icon/lg/arrow-down';
+import BpkText, {
+  WEIGHT_STYLES,
+  TEXT_STYLES,
+} from 'bpk-component-text/src/BpkText';
+
+import { LatLongPropType, type LatLong } from './common-types';
+import BpkBasicMapMarker from './BpkBasicMapMarker';
+import STYLES from './BpkPriceMarker.scss';
+
+const getClassName = cssModules(STYLES);
+
+export const MARKER_STATUSES = {
+  default: 'default',
+  focused: 'focused',
+  viewed: 'viewed',
+};
+
+type Props = {
+  label: string,
+  position: LatLong,
+  disabled: boolean,
+  className: ?string,
+  arrowClassName: ?string,
+  onClick: ?(event: SyntheticEvent<>) => mixed,
```

Comment:
Just checking, should `onClick` be optional? I made it required in RN because I figured it's like a type of button and should always have an action. Happy to discuss though, I could be wrong

Reply:
Let me check as I took this from the `BpkMapMarker.js` so might be from historical reasons. I guess it might have been that not always an action was needed and was used to just display but not sure the reasoning from when it was built

Reply:
Actually you might know as you built the component 😆

Reply:
![spider-man pointing at self meme](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fstatic2.cbrimages.com%2Fwordpress%2Fwp-content%2Fuploads%2F2019%2F03%2FSpider-Man-Pointing-Meme.jpg&f=1&nofb=1)

Yeah let's just leave it optional then, as I guess it'll be weird if it's optional in `BpkMapMarker` but required in `BpkPriceMarker`.